src/data/list/rotate.lean

```diff
@@ -3,10 +3,10 @@ Copyright (c) 2019 Chris Hughes. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Chris Hughes
 -/
-import data.list.basic
+import data.list.perm
+import data.list.range
 
-universes u
-variables {α : Type u}
+variables {α : Type*}
```

Comment:
Is there a reason to remove the explicit universe annotations?

Reply:
Just a style change. Can easily revert it.